gogatekeeper.yaml

```diff
@@ -0,0 +1,65 @@
+package:
+  name: gogatekeeper
+  version: "3.3.0"
+  epoch: 0
+  description: An OpenID / Proxy service
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - go
+  environment:
+    CGO_ENABLED: "0"
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 2cca2d0c6a8999fbca1b428cc500a34ca8d472f3
+      repository: https://github.com/gogatekeeper/gatekeeper
+      tag: ${{package.version}}
+
+  - uses: go/bump
+    with:
+      deps: |-
+        golang.org/x/oauth2@v0.27.0
+        github.com/open-policy-agent/opa@v1.4.0
+
+  - uses: go/build
+    with:
+      ldflags: |
+        -X github.com/gogatekeeper/gatekeeper/pkg/proxy/core.release=${{package.version}}
+        -X github.com/gogatekeeper/gatekeeper/pkg/proxy/core.gitsha=$(git rev-parse --short HEAD)
+        -X github.com/gogatekeeper/gatekeeper/pkg/proxy/core.compiled=$(date -d@${SOURCE_DATE_EPOCH} +%s)
+      output: gatekeeper
+      packages: ./cmd/keycloak/gatekeeper-keycloak.go
+
+subpackages:
+  - name: ${{package.name}}-compat
+    description: gatekeeper compat
+    pipeline:
+      - runs: |
+          # Symlink the binary from usr/bin to /opt/gatekeeper
+          mkdir -p "${{targets.contextdir}}"/opt/gatekeeper
+          ln -sf /usr/bin/gatekeeper ${{targets.contextdir}}/opt/gatekeeper/gatekeeper
+    test:
+      environment:
+        contents:
+          packages:
+            - ${{package.name}}
+      pipeline:
+        - runs: test "$(readlink /opt/gatekeeper/gatekeeper)" = "/usr/bin/gatekeeper"
+
+update:
+  enabled: true
+  github:
+    identifier: gogatekeeper/gatekeeper
+
+test:
+  pipeline:
+    - runs: |
+        gatekeeper --help
+        gatekeeper --version | grep -q "${{package.version}}"
```

Comment:
Is it possible to expand the tests with `test/daemon-check-output`? If not, a comment would be great to have here.

Reply:
its hard to test its functionality in melange tests since it needs keycloak so this is why I keep tests small here.